interfaces/IOCDM.h

```diff
@@ -392,6 +392,29 @@ class EXTERNAL KeyId {
     ISession::KeyStatus _status;
 };
 
+// @stubgen:include <com/IIteratorType.h>
+// @json 1.0.0
+struct EXTERNAL IOCDM : virtual public Core::IUnknown {
+    enum { ID = ID_OCDM_INFO };
+
+    virtual ~IOCDM() = default;
+
+    using IStringIterator = RPC::IIteratorType<string, RPC::ID_STRINGITERATOR>;
+    struct Drm {
+        string name /* @brief Name of the DRM */;
+        string /* @opaque */ keySystems /* @brief Supported key Systems */;
+    };
+    using IDrmIterator = RPC::IIteratorType<Drm, ID_OCDM_DRMS_ITERATOR>;
+
+    // @alt Drms
+    // @property
+    // @brief Systems - Retrieves all key systems available in the system (e.g. Nagra, PlayReady, WideVine etc)
+    virtual Core::hresult Systems(IDrmIterator*& drms /* @out */) const = 0;
```

Comment:
I think this returns a list of "keySystems", so I think this should be:

virtual Core::hresult Systems(IStringIterator*& keySystems /* @out */) const = 0;

The Drm struct is not needed at all...